learn/tanssi/included-templates.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Appchain Templates Included in Tanssi
 description: Tanssi protocol provides useful templates to start building your appchain, including a ready-to-use EVM template for Ethereum compatibility.
+icon: material-developer-board
```

Comment:
I like this icon! But it doesn't feel cohesive with the other icons. Maybe this one can be used instead `:octicons-copy-24:`